src/meta_memcache/interfaces/router.py

```diff
@@ -15,6 +15,19 @@
 )
 
 
+class FailureHandling(NamedTuple):
+    """
+    Override the default failure handling for a execution
+    """
+
+    # None means use the default for the pool
+    raise_on_server_error: Optional[bool] = None
+    track_write_failures: bool = True
```

Comment:
For `track_write_failures` there's no pool default? Otherwise shouldn't it be `Optional[bool]`?

Reply:
The pool always has a write failure tracker hook, but it might or might not be configured. If a pool doesn't want write failures, it can just not configure it. But if you think it will make this easier to understand here, I can also add it.